qt/scientific_interfaces/Indirect/IndirectDiffractionReduction.cpp

```diff
@@ -203,6 +217,7 @@ void IndirectDiffractionReduction::plotResults() {
   QString pyInput = "from mantidplot import plotSpectrum, plot2D\n";
```

Comment:
Seems confusing that this is declared outside of `#if QT_VERSION < QT_VERSION_CHECK(5, 0, 0)` when it is only used inside of that macro (from what I can tell).